modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/OrdersListView.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayLink from '@clayui/link';
+import React, {useRef} from 'react';
+
+import ServiceProvider from '../../../ServiceProvider/index';
+import OrdersTable from '../OrdersTable';
+import {VIEWS} from '../util/constants';
+import {composeFilterByAccountId} from '../util/index';
+import EmptyListView from './EmptyListView';
+import ListView from './ListView';
+
+const {baseURL: ORDERS_RESOURCE_ENDPOINT} = ServiceProvider.AdminOrderAPI('v1');
+
+function OrdersListView({
+	createOrderURL,
+	currentAccount,
+	selectOrderURL,
+	setCurrentView,
+}) {
+	const ordersListRef = useRef();
+
+	return (
+		<ClayDropDown.ItemList className={'orders-list-container'}>
+			<ClayDropDown.Section className={'item-list-head'}>
+				<ClayButtonWithIcon
+					displayType={'unstyled'}
+					onClick={() => setCurrentView(VIEWS.ACCOUNTS_LIST)}
+					symbol="angle-left-small"
+				/>
+
+				<span className={'text-truncate-inline'}>
+					<span className={'text-truncate'}>
+						{currentAccount.name}
+					</span>
+				</span>
+			</ClayDropDown.Section>
+
+			<ClayDropDown.Divider />
+
+			<ClayDropDown.Section className={'item-list-body'}>
+				<ListView
+					apiUrl={`${ORDERS_RESOURCE_ENDPOINT}?${composeFilterByAccountId(
+						currentAccount.id
+					)}`}
+					contentWrapperRef={ordersListRef}
+					customView={({items, loading}) => {
+						if (!items || !items.length) {
+							return (
+								<EmptyListView
+									caption={'no-orders-were-found'}
+									loading={loading}
+								/>
+							);
+						}
+
+						return (
+							<OrdersTable
+								orders={items}
+								selectOrderURL={selectOrderURL}
+							/>
+						);
+					}}
+					placeholder={'search-order'}
```

Comment:
@wincent 
For this whole family of corrections: the `prop={'...'}` is an automation of the IDE. Personally I prefer it as `props` are not `HTML attributes` and for code look-up/inspections the 2 distinct syntaxes do mark such a... distinction. But yet I'll convert it if it's so "disturbing". :D

Nonetheless, the whole of this component has basically stayed untouched from the version that came to my hands, save some code-splitting to allow for better testability and more fine-grained unit test coverage. It'll certainly undergo a deeper refactor.

Reply:
It's not "disturbing". It's just that we (Liferay) strongly value consistency so having mixed style within local regions of a file like you have in this PR is a no-go (Brian may outright reject it if he looks closely enough), and broader consistency across the entire code base is desirable too, regardless of preferences, so liferay-portal is overwhelmingly inclined towards `"x"` over `{'x'}` and I'd recommend you just go with that flow.

I wouldn't worry too much about it though as we will auto-enforce anyway so that people don't have to fret over it.